triplets/community/arm-android.cmake

```
@@ -0,0 +1,4 @@
+set(VCPKG_TARGET_ARCHITECTURE arm)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE dynamic)

```

```suggestion
set(VCPKG_LIBRARY_LINKAGE static)
```
Because we prefer static library linkage on Linux by default, we should do the same for android.

---

I agree.

---

I also want to do more changes to the android.cmake file.

---

@ras0219-msft What is VCPKG_CRT_LINKAGE? should it also be static?

---

`VCPKG_CRT_LINKAGE` is just for Windows.

---

Maybe I will remove `VCPKG_CRT_LINKAGE` from these triplets. 
I succeeded to compile rappture to armv6, but didn't succeeded to consume it (to compile this lib with another application).
When I will success, I will push the change you requested, and some more.
Thank you for your feedback.

---

@JackBoosY I update the PR.

---

@ras0219-msft I update like you request, please review my PR. thank you.